src/core/tsi/alts/crypt/gsec.h

```diff
@@ -20,6 +20,7 @@
 #define GRPC_CORE_TSI_ALTS_CRYPT_GSEC_H
 
 #include <grpc/support/port_platform.h>
```

Comment:
I believe our linter will complain that there is no space below the port_platform import.